src/components/shared/SelectRecipients/SelectRecipientsDialog.tsx

```diff
@@ -55,7 +57,22 @@ export function SelectRecipientsDialogUI(props: SelectRecipientsDialogUIProps) {
     const { t } = useI18N()
     const classes = useStylesExtends(useStyles(), props)
     const { items, disabledItems } = props
-
+    const [search, setSearch] = React.useState('')
+    const [, inputRef] = useCapturedInput((newText) => {
+        setSearch(newText)
+    }, [])
+    const searchFilter = React.useCallback(
+        (item: Profile) => {
```

Comment:
Unlike in `<SelectPeopleAndGroupsUI />`, the type of item is `PersonOrGroup `. Here item is just `Profile`, so removing the related logic to check it is a `Group` or `Person`